cmd/bee/cmd/db_test.go

```diff
@@ -179,6 +181,102 @@ func TestDBExportImportPinning(t *testing.T) {
 	}
 }
 
+func TestDBNuke(t *testing.T) {
+	t.Parallel()
+
+	dataDir := t.TempDir()
+	ctx := context.Background()
+	db := newTestDB(t, ctx, &storer.Options{
+		Batchstore:      new(postage.NoOpBatchStore),
+		RadiusSetter:    kademlia.NewTopologyDriver(),
+		Logger:          testutil.NewLogger(t),
```

Comment:
Does the test need to output the log messages? The same is below.

Reply:
The node needs a logger

Reply:
If you dont specify one it should use a noop logger

Reply:
well, it doesnt. It panics at 

```
panic: runtime error: invalid memory address or nil pointer dereference

logger := opts.Logger.WithName("epochmigration").Register()`
```

Also, It still logs even if I pass the noop logger in the options
```
&storer.Options{
   ...
  Logger: log.Noop,
}
```